packages/clerk-js/src/ui/components/KeylessPrompt/index.tsx

```
@@ -50,29 +97,25 @@ const _KeylessPrompt = (_props: KeylessPromptProps) => {
     return url.href;
   }, [claimed, _props.copyKeysUrl, _props.claimUrl]);
 
-  const baseElementStyles = css`
-    box-sizing: border-box;
-    padding: 0;
-    margin: 0;
-    background: none;
-    border: none;
-    line-height: 1.5;
-    font-family:
-      -apple-system,
-      BlinkMacSystemFont,
-      avenir next,
-      avenir,
-      segoe ui,
-      helvetica neue,
-      helvetica,
-      Cantarell,
-      Ubuntu,
-      roboto,
-      noto,
-      arial,
-      sans-serif;
-    text-decoration: none;
-  `;
+  const instanceUrlToDashboard = useMemo(() => {
+    try {
+      const url = new URL(
+        `${redirectUrlParts.baseDomain}/apps/${redirectUrlParts.appId}/instances/${redirectUrlParts.instanceId}/user-authentication/email-phone-username`,
+      );
+      return url.href;
+    } catch {
+      return '';
```

Wouldn't it be better to use the last-active path as fallback instead of an empty string?

---

I see 👍 I guess my point was that after `copyKeysUrl` is malformed and then also the regex fails, should we have a fallback that "does something" (because empty string won't do anything and the user also doesn't know what's wrong) or should we throw an error? We probably don't want to show an error since it's not actionable (they can't do anything about it) but silently failing is also not nice.

Not familiar with all moving pieces here so I'll trust your judgement :)

---

Let's having something more solid and then iterate. I've pushed [this](https://github.com/clerk/javascript/pull/5013/commits/5872d9faf958e83c50ecaee1afad7518c9b7091c)